integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -293,6 +229,20 @@ private void checkDomainRollingRestarted() throws Exception {
     }
   }
 
+  /**
+   * Check whether the weblogic server instances are still RUNNING 
```

Comment:
none of the javadoc in this file is javadoc compliant.  can you please run javadoc and fix the errors?

Reply:
Does the style check plugin does not check for javadoc compatibility. When I run "javaoc  ItOperatorUpgrade.java" It only complains about the missing package 
--
ItOperatorUpgrade.java:19: error: package org.junit.jupiter.api does not exist
import org.junit.jupiter.api.AfterEach;
---

Reply:
i get this error: 
```
/home/mark/jdk-11.0.6/bin/javadoc -private -splitindex -d /home/mark/operator/weblogic-kubernetes-operator/integration-tests/target/site @/tmp/javadoc_args
Loading source file /home/mark/operator/weblogic-kubernetes-operator/integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java...
Constructing Javadoc information...
Standard Doclet version 11.0.6
Building tree for all the packages and classes...
Generating /home/mark/operator/weblogic-kubernetes-operator/integration-tests/target/site/oracle/kubernetes/operator/ItOperatorUpgrade.html...
/home/mark/operator/weblogic-kubernetes-operator/integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java:141: error: @param name not found
   * @param restart boolean parameter used to determine if a restart of domain is checked
            ^
```

the ide shows it too:
![Screenshot from 2020-03-23 11-44-31](https://user-images.githubusercontent.com/6345393/77452756-afe50800-6dcc-11ea-90ea-307a97e142f6.png)

Reply:
![Screenshot from 2020-03-24 12-38-35](https://user-images.githubusercontent.com/6345393/77452860-d2772100-6dcc-11ea-8a53-2b47757c2364.png)

Reply:
Please spell "WebLogic" right too

Reply:
Replaced  WebLogic --> Weblogic

Reply:
@anpanigr, the product name has the capital L.  The correct name is "WebLogic"

Reply:
i cant see the update - still appears to say `weblogic`

Reply:
Modified in latest commit